lang/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+###############################################################################
+## Copyright (C) 2009-2021 Arzel Jérôme <myst6re@gmail.com>
+## Copyright (C) 2020 Julian Xhokaxhiu <https://julianxhokaxhiu.com>
+##
+## This program is free software: you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation, either version 3 of the License, or
+## (at your option) any later version.
+##
+## This program is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+## GNU General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with this program.  If not, see <http://www.gnu.org/licenses/>.
+###############################################################################
+
+find_package(QT NAMES Qt5 COMPONENTS LinguistTools)
+find_package(Qt${QT_VERSION_MAJOR}LinguistTools)
+
+file(GLOB_RECURSE TS_FILES "${RELEASE_NAME}_*.ts")
+
+qt5_create_translation(QM_FILES ${CMAKE_SOURCE_DIR} ${TS_FILES})
```

Comment:
I haven't done the install part yet. What's the target? Is it a custom command like `cmake translations`?

Reply:
This is basicly saying . we have a new target named
Translations it required by ALL other targets and for this to be complete it DEPENDS on the ${QM_FILES}
So that any build of anything will trigger this. 

Then later when we do an install this will install the following files. install rules are created if the target is  built